packages/docs/jest.config.js

```diff
@@ -19,6 +19,9 @@ module.exports = {
   setupFiles: ['./src/fetch-polyfill.js'],
   setupFilesAfterEnv: ['./src/setupTests.js'],
   testEnvironment: 'jsdom',
+  testEnvironmentOptions: {
+    customExportConditions: [''],
```

Comment:
We must use node-like module resolution in JSDOM. JSDOM uses a browser module resolution by default and that's a rather big oversight on their part.